### PR TITLE
Alerting: Cleanup template, silence and notification files created du…

### PR DESCRIPTION
**Pull request by santihernandezc**
**What this PR does / why we need it**:
Deletes the parent directory of all files created during tests.

**Which issue(s) this PR fixes**:
[#38907](https://github.com/grafana/grafana/issues/38907)

### CODE REVIEW

**Review by kylebrandt — APPROVED** [review]
Looks okay to me, but should have the error checked (maybe just log on failure?) as the linter says.

**Review by grobinson-grafana — APPROVED** [review]

File: pkg/services/ngalert/notifier/multiorg_alertmanager_test.go [hunk 1]
```diff
@@ -18,11 +20,19 @@ func TestMultiOrgAlertmanager_SyncAlertmanagersForOrgs(t *testing.T) {
 	orgStore := &FakeOrgStore{
 		orgs: []int64{1, 2, 3},
 	}
+
+	tmpDir, err := ioutil.TempDir("", "test")
+	if err != nil {
+		t.Fatalf("ioutil.TempDir() err = %v", err)
+	}
```
Comment (papagian) [inline pkg/services/ngalert/notifier/multiorg_alertmanager_test.go:27]: Alternatively you could use `require.NoError(t, err)` but this is also fine.

**Review by papagian — APPROVED** [review]

File: docs/sources/administration/configuration.md [hunk 2]
```diff
@@ -471,9 +471,10 @@ Google Tag Manager ID, only enabled if you enter an ID here.
 ### application_insights_connection_string
 
 If you want to track Grafana usage via Azure Application Insights, then specify _your_ Application Insights connection string. Since the connection string contains semicolons, you need to wrap it in backticks (`). By default, tracking usage is disabled.
-		
+
 ### application_insights_endpoint_url
-		Optionally, use this option to override the default endpoint address for Application Insights data collecting. For details,  refer to the [Azure documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/app/custom-endpoints?tabs=js).
+
+    	Optionally, use this option to override the default endpoint address for Application Insights data collecting. For details,  refer to the [Azure documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/app/custom-endpoints?tabs=js).
```
Comment (achatterjee-grafana) [inline docs/sources/administration/configuration.md:477]:
```suggestion
    	Optionally, use this option to override the default endpoint address for Application Insights data collecting. For details, refer to the [Azure documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/app/custom-endpoints?tabs=js).
```

File: docs/sources/enterprise/access-control/fine-grained-access-control-references.md [hunk 3]
```diff
@@ -31,8 +31,8 @@ The reference information that follows complements conceptual information about
...
+| Admin         | `fixed:users:org:edit`<br>`fixed:users:org:read`<br>`fixed:reporting:admin:edit`<br>`fixed:reporting:admin:read`                                                                                                                                                                                                                                                                                                            | Allow access to the same resources and permissions that the [Organization Admin]({{< relref "../../permissions/organization_roles.md" >}}) has by default.                 |
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/access-control/fine-grained-access-control-references.md:37]:
```suggestion
| Admin         | `fixed:users:org:edit`<br>`fixed:users:org:read`<br>`fixed:reporting:admin:edit`<br>`fixed:reporting:admin:read`                                                                                                                                                                                                                                                                                                            | Allow access to the same resources and permissions that the [Grafana organization administrator]({{< relref "../../permissions/organization_roles.md" >}}) has by default.                 |
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc review.

**Pull request merged**